test/powershell/Modules/PowerShellGet/PowerShellGet.Tests.ps1

```diff
@@ -25,11 +25,14 @@ if(IsInbox)
 {
     $script:ProgramFilesPSPath = Microsoft.PowerShell.Management\Join-Path -Path $env:ProgramFiles -ChildPath "WindowsPowerShell"
 }
-elseif(IsCoreCLR){
-    if(IsWindows) {
+elseif(IsCoreCLR)
+{
```

Comment:
Don't use C# style for PowerShell scripts.
Please revert the brace style changes. Below too.

Reply:
Fixed